### ISSUE TITLE
[Bug]: Vibrancy not applied in fullscreen

### ISSUE THREAD

**Issue by Spiderpig86** [illixion/vscode-vibrancy-continued#37]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Using Vscode 1.71.2, when I change the window to use fullscreen mode (f11), vibrancy disappears. It appears again once it is no longer fullscreen.

### Expected Behavior

Vibrancy should still be applied on the window even in fullscreen mode.

### Steps To Reproduce

1. Hit `f11` or enter fullscreen on Vscode.

### Environment

```markdown
- OS: Windows 10
- VSCode: 1.71.2
- Extension: 1.1.9
```

**Comment by illixion** [comment]
> I think this functions as intended, Windows and MacOS disables vibrancy on full screen to improve focus

Can confirm, this is likely intentional Electron behavior, you could maximize the window as an alternative.

**Comment by EmreSURK** [comment]
Nope, it is not. 
Both Safari and Finder are still vibrant. Blur radius becomes very large but they are still being affected by the desktop wallpaper. You can see a slightly different colors for different wallpaper images.

<img width="256" alt="Screenshot 2022-12-08 at 4 28 08 PM" src="https://user-images.githubusercontent.com/9450430/206459849-c287a69b-77a0-4a34-97f2-6a4877a4f847.png">

<img width="256" alt="Screenshot 2022-12-08 at 4 28 29 PM" src="https://user-images.githubusercontent.com/9450430/206459920-5ffe8150-d166-4e37-8bce-6f1aaf62551a.png">


As an alternative, we may put theme color on the background. 
Here is an issue I opened for this:
https://github.com/illixion/vscode-vibrancy-continued/issues/47

**Comment by MrConnorKenway** [comment]
This is intended. If you look at `Default Dark.css`, you will find
```css
.monaco-workbench.fullscreen {
	background-color: #202020 !important; /* make the background-color non transparent when in fullscreen */
}
```

To reenable vibrancy in fullscreen mode, change it to
```css
.monaco-workbench.fullscreen {
	background-color: transparent !important;
}
```

**Comment by illixion** [comment]
@EmreSURK
> Nope, it is not. Both Safari and Finder are still vibrant. Blur radius becomes very large but they are still being affected by the desktop wallpaper.

I believe what you're seeing is not transparency, but rather wallpaper tinting, which is a macOS setting that is on by default. With that setting off, there is no transparency.

@MrConnorKenway
> This is intended. If you look at Default Dark.css, you will find

Good catch, though after checking this on macOS, it doesn't appear to actually enable transparency. Instead it just takes the background color average and applies that. I'll look into this later, but feel free to use the Vibrancy custom import feature in the meantime.

**Comment by EmreSURK** [comment]
@illixion Yes, apparently you are right. 
Good Point. 
I checked the image pixel colors with Digital Color Meter if there is a slight difference we can not release it. No, no different, only one color.

**Comment by minhiu** [comment]
> This is intended. If you look at `Default Dark.css`, you will find
> 
> ```css
> .monaco-workbench.fullscreen {
> 	background-color: #202020 !important; /* make the background-color non transparent when in fullscreen */
> }
> ```
> 
> To reenable vibrancy in fullscreen mode, change it to
> 
> ```css
> .monaco-workbench.fullscreen {
> 	background-color: transparent !important;
> }
> ```

I have a dumb question but how do we modify this css file in VSCode?

**Comment by illixion** [comment]
> > This is intended. If you look at `Default Dark.css`, you will find
> > ```css
> > .monaco-workbench.fullscreen {
> > 	background-color: #202020 !important; /* make the background-color non transparent when in fullscreen */
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > To reenable vibrancy in fullscreen mode, change it to
> > ```css
> > .monaco-workbench.fullscreen {
> > 	background-color: transparent !important;
> > }
> > ```
> 
> I have a dumb question but how do we modify this css file in VSCode?

You can use the import feature for that. Create a CSS file somewhere on the disk with your desired contents, copy the full path (on Windows, Shift+Right click then choose Copy as path) and add it to your VSCode config under Vibrancy settings as described here: https://github.com/illixion/vscode-vibrancy-continued#vscode_vibrancyimports

If you're on Windows, you'll also need to replace all backslashes with forward slashes.

**Comment by illixion** [comment]
Fixed in **v1.1.92** by removing that fullscreen opaque rule, as I've found it wasn't doing anything useful anymore.

Also, I've ensured that wallpaper tinting applies to VSCode in full-screen like Safari and Finder when **System Settings → Appearance → "Allow wallpaper tinting in windows"** is enabled (see #47)
